### PR TITLE
chore: generated CHANGELOG for origin/release/v50.16.0

### PR DESCRIPTION
**Pull request by lcampos**
Updates changelog for release 50.16.0

### CODE REVIEW

**Review by brpowell — APPROVED** [review]

**Review by sfsholden — APPROVED** [review]

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+
+## Fixed
+
+#### docs
+
+- Add Log Analyzer to [Recommended Extensions](https://developer.salesforce.com/tools/vscode/en/getting-started/recommended-extensions) ([PR #2911](https://github.com/forcedotcom/salesforcedx-vscode/pull/2911))
+
+- Add AdoptOpenJDK configuration sample for Linux in [Java Setup](https://developer.salesforce.com/tools/vscode/en/getting-started/java-setup) ([PR #2870](https://github.com/forcedotcom/salesforcedx-vscode/pull/2870)) - Contribution by [@renatoliveira](https://github.com/renatoliveira)
+
+- Include steps to get the JDK install path for Mac OS in [Java Setup](https://developer.salesforce.com/tools/vscode/en/getting-started/java-setup) ([PR #2910](https://github.com/forcedotcom/salesforcedx-vscode/pull/2910)) - Contribution by [@mikeflemingcfs](https://github.com/mikeflemingcfs)
```
Comment (ekapner) [inline packages/salesforcedx-vscode/CHANGELOG.md:17]:
```suggestion
- Include steps to get the JDK install path for MacOS in [Java Setup](https://developer.salesforce.com/tools/vscode/en/getting-started/java-setup) ([PR #2910](https://github.com/forcedotcom/salesforcedx-vscode/pull/2910)) - Contribution by [@mikeflemingcfs](https://github.com/mikeflemingcfs)
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+
+- Add missing permission step to [ISV Customer Debugger](https://developer.salesforce.com/tools/vscode/en/apex/isv-debugger/#configure-isv-customer-debugger) ([PR #2901](https://github.com/forcedotcom/salesforcedx-vscode/pull/2901))
+
+#### salesforcedx-vscode-lwc
+
+- Reduce extension size by 34% ([PR #2904](https://github.com/forcedotcom/salesforcedx-vscode/pull/2904))
```
Comment (ekapner) [inline packages/salesforcedx-vscode/CHANGELOG.md:23]: Although this is totally cool, seems a bit like under the hood details unless customers have complained and we're doing this work in response to that.

**Review by ekapner — COMMENTED** [review]
a couple of small comments

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 2 above)
Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md:23]: Agreed, I've added them cause performance is one of the big things customers are opening issues for. This shows that we're actively working on it.

**Pull request merged**